src/scopes.ts

```diff
@@ -559,11 +561,20 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
     type: ScopeType.Modify,
     products: [TranscendProduct.ConsentManager],
   },
+  [ScopeName.DeployTestConsentManager]: {
+    dependencies: [ScopeName.ViewConsentManager],
+    description:
+      // eslint-disable-next-line max-len
+      'Ability to publish changes to the test Consent Manager bundle. This changes the code contents of airgap.js and attempts to invalidate the CDN.',
+    title: 'Deploy Test Consent Manager',
+    type: ScopeType.Modify,
+    products: [TranscendProduct.ConsentManager],
+  },
   [ScopeName.DeployConsentManager]: {
```

Comment:
@michaelfarrell76 should this one includes the test scope too or should they be completely separate?

Reply:
For now I tried to make it so that prod includes perms to deploy test as well